ui/app/templates/rules.html

```diff
@@ -86,6 +86,10 @@ <h3 class="panel-title">
                 ng-click="openEditScheduledRuleChangeModal(rule)">
             Modify Pending Delete
         </button>
+        <button class="btn btn-info btn-xs"
```

Comment:
I think this button should appear to the left of the "Modify Pending..." button. It looks odd to have it shown after the orange button in one row, and before it in another: https://screenshotscdn.firefoxusercontent.com/images/2d94cf06-bb49-48a7-a606-ff1aac507546.png